spec/omniauth/strategies/headquarters_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+require 'omniauth/strategies/headquarters'
+
+module OmniAuth
+  module Strategies
+    describe Headquarters do
+      let(:request) { double('Request', params: {}, cookies: {}, env: {}) }
+
+      subject do
+        args = ['appid', 'secret', @options || {}].compact
+        OmniAuth::Strategies::Headquarters.new(*args).tap do |strategy|
+          allow(strategy).to receive(:request) {
```

Comment:
Any reason for this not to be a one-liner? If you do want to use a multi-line block, our guides say to use `do...end`.